core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/annotations/ImportDataSchema.kt

```diff
@@ -43,6 +44,9 @@ public annotation class ImportDataSchema(
     val enableExperimentalOpenApi: Boolean = false,
 )
 
+@Target(AnnotationTarget.CLASS)
+public annotation class DataSchemaSource(val source: String, val qualifier: String = SchemaReader.DEFAULT_QUALIFIER)
```

Comment:
I can guess what `source` does, but `qualifier` is unclear for me. Some comments would be nice, even though it's just a proof-of-concept

Reply:
I think I remember vaguely from your demo that this allowed to make distinctions of some kind. But I don't remember exactly without a small example